extensions/kafka/PublishKafka.cpp

```diff
@@ -376,10 +326,9 @@ void PublishKafka::onSchedule(core::ProcessContext& context, core::ProcessSessio
   logger_->log_debug("PublishKafka: Max Flow Segment Size [{}]", max_flow_seg_size_);
 
   // Attributes to Send as Headers
-  std::string value;
-  if (context.getProperty(AttributeNameRegex, value) && !value.empty()) {
-    attributeNameRegex_ = utils::Regex(value);
-    logger_->log_debug("PublishKafka: AttributeNameRegex [{}]", value);
+  if (const auto attribute_name_regex = context.getProperty(AttributeNameRegex)) {
```

Comment:
We used to reject empty property values here, but we no longer do (also at a few other places further down).  Is this intentional?

Reply:
good catch, https://github.com/apache/nifi-minifi-cpp/pull/1885/commits/5b8a9e84ecd1a067f39e51666f47fd98f79ba10c#diff-8c7db1de63f906d0cc51e10730a764ddc94d61e8a5bc8d24154d05af025afa09L340